parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1195,17 +1200,23 @@ listOfLinks ppe results = pure $ P.lines [
   prettyType Nothing = "❓ (missing a type for this definition)"
   prettyType (Just t) = TypePrinter.pretty ppe t
 
-showDiffNamespace :: forall v . Var v => PPE.PrettyPrintEnv -> OBD.BranchDiffOutput v Ann -> Pretty
-showDiffNamespace ppe d@OBD.BranchDiffOutput{..} =
-  P.sepNonEmpty "\n\n" . (`State.evalState` (0::Int)) . sequence $ [
+showDiffNamespace :: forall v . Var v
+                  => PPE.PrettyPrintEnv
+                  -> Path.Absolute
+                  -> Path.Absolute
+                  -> OBD.BranchDiffOutput v Ann
+                  -> (Pretty, NumberedArgs)
+showDiffNamespace ppe oldPath newPath OBD.BranchDiffOutput{..} =
+  let wrangleResult (p,(_n,args)) = (P.sepNonEmpty "\n\n" p, toList args) in
+  wrangleResult . (`State.runState` (1::Int, Seq.empty)) . sequence $ [
     if (not . null) updatedTypes
     || (not . null) updatedTerms
     || propagatedUpdates > 0
     || (not . null) updatedPatches
     then do
       prettyUpdatedTypes :: [Pretty] <- traverse prettyUpdateType updatedTypes
       prettyUpdatedTerms :: [Pretty] <- traverse prettyUpdateTerm updatedTerms
-      prettyUpdatedPatches :: [Pretty] <- traverse prettySummarizePatch updatedPatches
+      prettyUpdatedPatches :: [Pretty] <- traverse (prettySummarizePatch newPath) updatedPatches
```

Comment:
Choice of `newPath` vs `oldPath` here is a little smelly, but for the current render design style, we have to pick one.  I wrote up more in #1064.